src/audio/clip_manager.cpp

```diff
@@ -4,7 +4,7 @@
 #include "engine/core/string.h"
 #include "engine/lumix.h"
 #define STB_VORBIS_HEADER_ONLY
-#include "stb_vorbis.h"
+#include "stb_vorbis.cpp"
```

Comment:
This will make it compile with C++ compiler instead of C on Windows, hopefully nothing will break